svgnative/src/SVGDocumentImpl.cpp

```diff
@@ -973,6 +973,128 @@ void SVGDocumentImpl::RenderElement(const Element& element, const ColorMap& colo
     SVG_ASSERT(mVisitedElements.empty());
 }
 
+Rect SVGDocumentImpl::Bounds()
+{
+    SVG_ASSERT(mGroup);
+    // TODO: Should we fire an assertion or raise exception?
+    if (!mGroup)
+        return Rect{};
+    ExtractBounds(*mGroup);
+    Rect sum_bound;
+    for(auto const &bound: mBounds)
```

Comment:
```cpp
for (auto const& bound : mBounds)
```

Don't we have styling enabled for testing?

Reply:
Styling as in CSS style or code style checks? We don't have a clang-tidy file or something like that, do we?

Reply:
I don't think our pipeline has any code styling checks, does it? In my case it's always failing because of the CMake problem I mentioned. I am using FetchContent which is not available in stretch. It needs buster.